pkg/store/bucket.go

```diff
@@ -1261,7 +1303,25 @@ func (s *BucketStore) LabelValues(ctx context.Context, req *storepb.LabelValuesR
 		}
 	}
 
+	// If we have series matchers, add <labelName> != "" matcher, to only select series that have given label name.
+	if len(reqSeriesMatchers) > 0 {
+		m, err := labels.NewMatcher(labels.MatchNotEqual, req.Label, "")
```

Comment:
We can make it a global variable with `labels.MustNewMatcher()`, WDYT? @pstibrany

Reply:
I don't quite follow what you mean. This cannot be a global variable, as it uses label name from the request. What am I missing?

Reply:
Sorry, I miss it. You are right.